mat64/io_test.go

```diff
@@ -276,3 +276,67 @@ func TestVectorIORoundTrip(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkMarshalDense10(b *testing.B)    { marshalBinaryBenchDense(b, 10) }
+func BenchmarkMarshalDense100(b *testing.B)   { marshalBinaryBenchDense(b, 100) }
+func BenchmarkMarshalDense1000(b *testing.B)  { marshalBinaryBenchDense(b, 1000) }
+func BenchmarkMarshalDense10000(b *testing.B) { marshalBinaryBenchDense(b, 10000) }
+
+func marshalBinaryBenchDense(b *testing.B, size int) {
+	m := NewDense(1, size, make([]float64, size))
```

Comment:
it shouldn't: we don't do any zero-suppression and we don't use var-ints.
I could modify the `marshalBinaryBench{Dense,Vector}` to generate some randome data, though.